src/Facility.LanguageServer/Program.cs

```diff
@@ -1,18 +1,18 @@
-using Facility.Definition;
 using Facility.LanguageServer;
-using Microsoft.Extensions.Logging;
```

Comment:
Do we still need the `Microsoft.Extensions.Logging` package reference?

Reply:
This builds but there is no logging. The current example uses Serilog on the Omnisharp repo. We could just copy that.

Reply:
Where do the logs go? I'd be more inclined to use `Microsoft.Extensions.Logging.Console`.

Reply:
OTOH, I don't think logs went anywhere before, and I don't know that there's anything being logged we'd care to see, so we could leave it unconfigured.

Reply:
right it didn't see like we were keeping track of that. I added https://github.com/FacilityApi/FacilityLanguageServer/pull/9/commits/2be83828e896a28fc41f0fc103c12f8217a66ebb as an example. Might need further configuration to be practical. But happy to revert or improve this.